README.md

```diff
@@ -1 +1,21 @@
-# elasticsearch-admin-scripts
\ No newline at end of file
+# elasticsearch-admin-scripts
+
+This repo stores the Cerebro team's scripts that make it easier to manage Elasticsearch. Detailed descriptions are in the scripts themselves.
+
+## Example usage
+```bash
+./createIndex.sh --user "username" --env "local" --index "myIndex" --docMappingFile "document_mapping.json"
+```
+
+Every script requires two arguments:
+* `env`: the environment of the cluster the script is run against (e.g. `oscf-dev`)
```

Comment:
Can we either:
1. list out all the current environment qualifiers
OR
2. talk about how that builds the URL (like where in the URL that piece goes) and then point to the config where the URLs are determined?